src/utils/sshUtils.ts

```diff
@@ -6,22 +6,23 @@
 import * as fse from 'fs-extra';
 import * as os from "os";
 import { join } from 'path';
+import { callWithMaskHandling } from 'vscode-azureextensionui';
 import { ext } from '../extensionVariables';
 import { localize } from '../localize';
 import { VirtualMachineTreeItem } from '../tree/VirtualMachineTreeItem';
 import { cpUtils } from "./cpUtils";
 
 export const sshFsPath: string = join(os.homedir(), '.ssh');
 
-export async function getSshKey(vmName: string): Promise<string> {
+export async function getSshKey(vmName: string, passphrase: string = ''): Promise<string> {
```

Comment:
It's a small thing, but I don't like `passphrase` defaulting to empty string here. I think "Secure by default" should apply to our internal coding style, meaning passphrase is a required parameter on this function

Reply:
How would I handle the case where the user doesn't type in a password?

Reply:
You just pass in an empty string...

Reply:
...oh, I understand what you mean now 😶